src/main/plugins/reminder-plugin/reminder-plugin.ts

```diff
@@ -0,0 +1,94 @@
+import {ExecutionPlugin} from "../../execution-plugin";
+import {PluginType} from "../../plugin-type";
+import {SearchResultItem} from "../../../common/search-result-item";
+import {TranslationSet} from "../../../common/translation/translation-set";
+import {UserConfigOptions} from "../../../common/config/user-config-options";
+import {ReminderOptions} from "../../../common/config/reminder-options";
+import {defaultReminderIcon} from "../../../common/icon/default-icons";
+import {join} from "path";
+// @ts-ignore
```

Comment:
Can you explain why you added the `@ts-ignore` here?

Reply:
withouth that line i get:
`TS7016: Could not find a declaration file for module 'parse-reminder'. 'C:\Users\USERNAME\OneDrive\Desktop\ueli\node_modu
les\parse-reminder\index.js' implicitly has an 'any' type.
`
Do you know how to fix this? I tried googling but I seem unable to get a solution.
It is my first time using Typescript so i dont know the inside out this errors

Changing to `import {parseReminder} from "parse-reminder";` still gives me the same error